openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -49,4 +47,4 @@ def handle(self, *args, **options):
             if not course_keys:
                 log.fatal('No courses specified.')
```

Comment:
I get this code was not changed in this or, so this isn't a blocker.

i just mean I'd image the logging and raising of an error to have a similar message and similar stack trace, rather than having the error hopefully happen at some future time and hopefully providing a reasonable message. Again, I can't see all code, so sorry if I'm missing something.